runtime/near-vm-runner/src/near_vm_runner.rs

```diff
@@ -533,59 +533,144 @@ impl Wasmer2VM {
     }
 }
 
-impl wasmer_vm::Tunables for &Wasmer2VM {
+impl near_vm_vm::Tunables for &NearVmVM {
```

Comment:
Similarly for this module.

Reply:
So this is a separate crate, that was copy-pasted from wasmer_vm ; if we rename it it’ll collide with the main near_vm crate (that was copy-pasted from the wasmer crate).

So yes, but it’ll have to wait until we merge the near-vm crates all together, as the crate split doesn’t make much sense for us